assets/styles/scss/header.scss

```diff
@@ -0,0 +1,72 @@
+/* Mixin for block 60%*/
+@mixin block60 {
+    width: 60%;
+    margin: auto;
+    max-width: 1200px;
+}
+/* Mixin for flexbox bloch row*/
+@mixin flexblock {
+    display: flex;
+    justify-content: space-between;
+}
+/* Colors for backgraund*/
+$bgcolorprimary: rgb(29, 29, 29);
```

Comment:
please move all colors into separate file variables.scss or colors.scss

Reply:
Add new files colors.scss and base.sccs